### ISSUE TITLE
Multipart Form Tab

### ISSUE THREAD

**Issue by VaslD** [ProxymanApp/Proxyman#833]
### Proxyman version? (Ex. Proxyman 1.4.3)

Version 2.20.0 (22000)

### macOS Version? (Ex. mac 10.14)

macOS Big Sur

Version 11.2.3 (20D91)

### Steps to reproduce

Call any API with Multipart Form for arbitrary binary files.

### Expected behavior

Would be really nice if there's a Multipart Form tab like Form that shows entry name and file pairs, preferably with individual files available for export/download.

Or better just a human-readable section that basically pretty-prints the whole binary mess.

Something like:

Part Name | Content Type | Value
-- | -- | --
image | image/jpeg | [Binary] ← Clickable for export

Instead of:

```
--Boundary+FC8B3E60558D113C
Content-Disposition: form-data; name="image"; filename="IMG_0127.JPG"
Content-Type: image/jpeg

ÿØÿàJFIFØØÿáExifMM*JR(iZØØ  ú üÿí8Photoshop 3.08BIM8BIM%ÔÙ²é	ìøB~ÿÀüú"ÿÄ	
ÿÄµ}!1AQa"q2¡#B±ÁRÑð$3br	
%&'()*456789:CDEFGHIJSTUVWXYZcdefghijstuvwxyz¢£¤¥¦§¨©ª²³´µ¶·¸¹ºÂÃÄÅÆÇÈÉÊÒÓÔÕÖ×ØÙÚáâãäåæçèéêñòóôõö÷øùúÿÄ
```

I know we have a Body tab that shows raw bytes as well as text for Multipart Form, but that's far from human-readable.

**Comment by NghiaTranUIT** [comment]
Hey @VaslD, can you check out this build: https://github.com/ProxymanApp/Proxyman/issues/827#issuecomment-803560921

We support multipart/form-data as you like 👍

**Comment by NghiaTranUIT** [comment]
Ah, I understand what you mean. You would like to export each part of the multipart/form-data

**Comment by VaslD** [comment]
> 
> 
> Ah, I understand what you mean. You would like to export each part of the multipart/form-data

Yes. I see that you use Paw as well. Basically I'd like to have multipart form requests shown somewhat like in the API builder view of Paw, not the response view. Sometimes I have to capture real traffic from my iOS app, so Paw isn't always available.

This way it'll be easier to spot tiny key-value pairs (like "Content-Disposition: form-data; name: count; 25") in a sea of hexes. Also the ability to export (or even copy-paste sounds fine) each part of the Form makes debugging file integrity and obscure server errors like "Unknown Media Container" easier.

Thank you.

**Comment by NghiaTranUIT** [comment]
It's possible to implement it since the code I prepare to parse the multipart/form-data is already finished in https://github.com/ProxymanApp/Proxyman/issues/827#issuecomment-803560921

I will support this feature in next releases 🙌

**Comment by NghiaTranUIT** [comment]
Hey @VaslD I've implemented this feature. If you don't mind, please try this beta build: https://proxyman.s3.us-east-2.amazonaws.com/beta/Proxyman_2.26.0_Multipart_Form_Data_Previewer.dmg

- You can preview the multipart/form-data with a nice table view
- Export each part as a file 👍 

<img width="1978" alt="Screen_Shot_2021-05-25_at_15_09_07" src="https://user-images.githubusercontent.com/5878421/119465504-ea9a5580-bd6d-11eb-9d4d-3725a37f09ec.png">

**Comment by VaslD** [comment]
Sorry for the delay. I’ve been busy this week and hadn’t had the time to try the Beta. I’ll see if I can upgrade my copy tomorrow and give it a go.

The screenshot looks promising though. It’s exactly what I wanted and I think this addition will be very crucial to my workflow.

Thank you again for making it a reality.

- Yi
